packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1799,10 +1799,16 @@ abstract class SingleChildRenderObjectWidget extends RenderObjectWidget {
   SingleChildRenderObjectElement createElement() => SingleChildRenderObjectElement(this);
 }
 
-/// A superclass for RenderObjectWidgets that configure RenderObject subclasses
+/// A superclass for [RenderObjectWidget]s that configure [RenderObject] subclasses
 /// that have a single list of children. (This superclass only provides the
 /// storage for that child list, it doesn't actually provide the updating
 /// logic.)
+///
+/// This will typically return a [RenderBox] mixing in [ContainerRenderObjectMixin]
```

Comment:
Mixing in `ContainerRenderObjectMixin` is required (see, e.g., `MultiChildRenderObjectElement.insertChildRenderObject`, which unconditionally casts the render object).

Reply:
The "typically" was referring to the `RenderBox` part. I will see how I can rewrite it.